app/src/main/java/org/apache/taverna/mobile/data/model/Author.java

```diff
@@ -6,9 +24,7 @@
 import android.os.Parcel;
 import android.os.Parcelable;
 
-/**
- * Created by Sagar
- */
+
 public class Author implements Parcelable {
 
     public static final Parcelable.Creator<Author> CREATOR = new Parcelable.Creator<Author>() {
```

Comment:
POJO objects should be first and then getter setter and at last Parcelable code.